pkg/cli/util.go

```diff
@@ -0,0 +1,78 @@
+package cli
+
+import (
+	"net"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kurl/pkg/netutils"
+	"github.com/spf13/cobra"
+)
+
+func newUtilCommand(cli CLI) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "util",
+		Short: "Utility commands",
```

Comment:
My concern here is that anything could fit under this command
Is this feature something that is valid for other cases? if not could we have a flannel command and some subcommand to it like `flannel find-private-interface`? WDYT?

Reply:
ive changed this to netutil